app/views/content_items/fatality_notice.html.erb

```diff
@@ -0,0 +1,46 @@
+<%= content_for :page_class, @content_item.format.dasherize %>
+<%= content_for :title, "#{@content_item.page_title} - #{t("content_item.format.#{@content_item.document_type}", count: 1)}" %>
+<%= content_for :meta_description, @content_item.description %>
+
+<div class="grid-row">
+  <div class="column-two-thirds">
+    <%= render 'govuk_component/title',
+        context: "Operations in #{@content_item.field_of_operation.title}",
+        title: @content_item.title %>
+  </div>
+</div>
+
+<%= render 'shared/withdrawal_notice', content_item: @content_item %>
+
+<div class="grid-row">
+  <div class="column-two-thirds">
+    <%= render 'govuk_component/metadata',
+        from: @content_item.from,
+        first_published: @content_item.published,
+        last_updated: @content_item.updated,
+        see_updates_link: true,
+        other: {
+          "Field of operation" => link_to(@content_item.field_of_operation.title, @content_item.field_of_operation.path)
+        }
+      %>
+  </div>
+</div>
+
+<%= render 'shared/description', description: @content_item.description %>
+
+<div class="grid-row sidebar-with-body">
+  <div class="column-third">
+    <%= render 'shared/sidebar_image',
+        image: { "url" => asset_path("ministry-of-defence-crest"), "alt_text" => "" } %>
```

Comment:
We've done this on purpose to let assistive tech know that this image is presentational.

https://davidwalsh.name/accessibility-tip-empty-alt-attributes

We should add a comment explaining this since it's not an obvious thing.